src/qae/object-properties.js

```diff
@@ -26,8 +26,6 @@ const properties = {
     qMeasures: [],
     /** @type {number[]} */
     qColumnOrder: [],
-    /** @type {number[]} */
```

Comment:
This would be a breaking. But I assume that we don't need and/or use it anymore?

Reply:
we never used it, so I think we would be fine

Reply:
it turns out that columnWidths is used in https://github.com/qlik-oss/sn-table/blob/main/src/ext/data.js#L45